src/components/ApiTable/ApiTable.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
```

Comment:
create a directory name common and put this apiTable in that directory

Reply:
done @IshfaqAhmedAwan